lightwood/mixer/regression.py

```diff
@@ -41,10 +63,24 @@ def fit(self, train_data: EncodedDs, dev_data: EncodedDs) -> None:
         log.info(f'Regression based correlation of: {self.model.score(X, Y)}')
 
     def partial_fit(self, train_data: EncodedDs, dev_data: EncodedDs) -> None:
```

Comment:
The inputs to partial-fit are incorrectly named. We should change it to `old_data` and `new_data`. I suspect this is a holdover of train's inputs, where dev data is specifically reserved for hyperparam/earlystopping etc.

In this case, the name "dev_data" is actually used to adjust the model with new information; this is *unintuitive*. We need to fix this for the entire codebase.